src/lib/requests/createSigningApiUrl/index.ts

```diff
@@ -0,0 +1,6 @@
+const API_VERSION = "v3";
+
+export const createSigningApiUrl = (resource: "signin" | "signup"): string =>
```

Comment:
Nice, that makes it a bit more obvious that the `NEXT_PUBLIC_TOKEN_API_URL` is in fact the same one. We could also think about renaming the Env Var to something more appropriate that includes all its functionalities.

Maybe consider fixing the mini typo here and wherever it's used: `createSigningApiUrl` -> `createSigninApiUrl`

Reply:
Thx! Yes it's true I thought about this. I'll make a suggestion. We have to remember updating the vars in vercel and 1password.

Reply:
I intentionally called the function `createSigningApiUrl` as it doesn't only produces urls for sign **in** but also for sign **up**. But maybe there is something less confusing?

Reply:
> I intentionally called the function createSigningApiUrl

Ahhh, got it... In that case, I am fine with leaving it the way it is. One alternative could be `createAuthApiUrl`?

Reply:
I now have removed the `createSigningApiUrl ` function entirely and refactored the different functions:
- `createSupabaseUrl` -> Former `createApiUrl`
- `createApiUrl` -> Now leads to our API

Reply:
Great, this is much more straightforward!